ocs/ocp.py

```diff
@@ -66,11 +66,12 @@ def exec_oc_cmd(self, command):
         out = run_cmd(cmd=oc_cmd)
         return munchify(yaml.safe_load(out))
 
-    def get(self, resource_name='', out_yaml_format=True):
+    def get(self, selector='', resource_name='', out_yaml_format=True):
```

Comment:
should be implemented as part of https://github.com/red-hat-storage/ocs-ci/pull/70/files#diff-9d7ae45bb7efabea0529c53dbca720f9R70